.github/workflows/agentics-maintenance.yml

```
@@ -0,0 +1,614 @@
+on:
+  schedule:
+    - cron: "37 0 * * *"  # Daily (based on minimum expires: 30 days)
+  workflow_dispatch:
+    inputs:
+      operation:
+        description: 'Optional maintenance operation to run'
+        required: false
+        type: choice
+        default: ''
+        options:
+          - ''
```

actionlint (1.7.12)

[error] string should not be empty

(syntax-check)